Makefile

```diff
@@ -53,51 +65,18 @@ update_circleci_reference: test
 #build_32bit: build_environment
 #	COMPILED_TAG_NAME=32bit COMPILE_OPTION=32BIT=Y $(MAKE) build
 
-build_debug: build_environment
-	COMPILED_TAG_NAME=debug COMPILE_OPTION="REPRO=\\\nDEBUG=Y" $(MAKE) build
-
-# We might *have* serialized data (pulled from cloud) and not want to rebuild the model...
-build_environment_serialize: build_environment
```

Comment:
I was hoping to call the 'build_environment' and 'build_compiled' targets here and just set different constants for e.g. ENVIRONMENT_TAG_NAME, but because the Serialbox docker targets are in the same file as the default ones I can no longer reuse these in the same way. e.g. if I set the ENVIRONMENT_TARGET  to target fv3gfs-environment-serialbox, it builds the image us.gcr.io/vcm-ml/fv3gfs-environment-serialbox:serialize. Would you mind changing ENVIRONMENT_IMAGE=$(GCR_URL)/$(ENVIRONMENT_TARGET):$(ENVIRONMENT_TAG_NAME) to a "?=" so I have the option to set that externally?

Reply:
You would do this by running `ENVIRONMENT_TARGET=fv3gfs-environment-serialibox make build_environment`. `ENVIRONMENT_IMAGE` is constructed from the target and tag names. This would create `us.gcr.io/vcm-ml/fv3gfs-environment-serialbox:latest`. You could rename the image if you like using `docker tag us.gcr.io/vcm-ml/fv3gfs-environment-serialbox:latest <new_name>`.

It's a bit of a name conflict to name the fv3gfs-environment-serialbox target `fv3gfs-environment` because that is the name of the "model build" environment. Even when the serialize model is built, it does not include anything installed in `fv3gfs-environment-serialbox` and is instead based off of `fv3gfs-environment`. The serialbox environment is used only to build Serialbox (as its build requirements are significantly greater than its runtime requirements) and to preprocess the sources. I've confirmed the source preprocessing works in the final image, so runtime source preprocessing and data generation does not require the dependencies installed in `environment-serialbox`.